app/experimenter/nimbus-ui/src/components/PageEditBranches/index.tsx

```diff
@@ -0,0 +1,19 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from "react";
+import { RouteComponentProps } from "@reach/router";
+import AppLayoutWithSidebar from "../AppLayoutWithSidebar";
+
+type PageEditBranchesProps = {} & RouteComponentProps;
+
+const PageEditBranches = (props: PageEditBranchesProps) => (
+  <AppLayoutWithSidebar>
+    <section data-testid="PageEditBranches">
```

Comment:
Nit: it looks like we already use kebab-case pretty widely

Reply:
I could change them across the board if we prefer, I was copying what we already had in `PageHome` and `PageNew` but that would WFM

Reply:
FWIW, I was using component class names for the testid's here for lazy consistency in tests - i.e. the component name & testid are copypasta same everywhere.

Reply:
Hmm actually, sort of tempted to do this in a follow up PR if this is the route we want to go. I started doing this but it touches a lot of files not related to this PR and Les has a couple `mockComponent` files setup that set `data-testid`s for us.

Reply:
Works for me so long as we're aiming for consistency!